Issue 10898 (#11224)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

You will most likely not need to specify any parameters except for `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API]({{site.url}}{{site.baseurl}}/api-reference/snapshots/create-repository/).

### HDFS

To use HDFS as a snapshot repository, follow these steps:

1. Create an HDFS directory for snapshots and ensure that the OpenSearch user has read and write permissions to it (ex: `/opensearch/repositories/searchable_snapshots`).

1. Install the `repository-hdfs` plugin on all nodes:

   ```bash
   sudo ./bin/opensearch-plugin install repository-hdfs
   ```

>    If you're using the Docker installation, see [Working with plugins]({{site.url}}{{site.baseurl}}/opensearch/install/docker/#working-with-plugins}). Your `Dockerfile` should look something like this:

Reviewer: ```suggestion
   If you're using the Docker installation, see [Working with plugins]({{site.url}}{{site.baseurl}}/opensearch/install/docker/#working-with-plugins}). Your `Dockerfile` should look similar to the following:
```

   ```
   FROM opensearchproject/opensearch:{{site.opensearch_version}}

   RUN /usr/share/opensearch/bin/opensearch-plugin install --batch repository-hdfs
   ```

1. (Optional) If you have Kerberos authentication enabled on your HDFS cluster:

>     If using Kerberos you may need to distribute the keytab file to all nodes and ensure that the OpenSearch user has read access to it. Otherwise, skip this step.

Reviewer: ```suggestion
```

1. Restart all nodes in the OpenSearch cluster.

1. Register the repository using the OpenSearch Snapshot API:

    ```json
    PUT _snapshot/searchable_snapshots
    {
      "type": "hdfs",
      "settings": {

>         // If using kerberos authentication, uncomment and replace YOURREALM with your realm name

Author: Agreed, thank you @jed326 @kolchfa-aws. Will commit the change in a moment.
